Update readmes to match common design (Local Server category) (#461)
Esri/arcgis-runtime-samples-java: local_server/local-server-geoprocessing/README.md

> # Local server geoprocessing

Reviewer: ```suggestion
# Local Server geoprocessing
```

Create contour lines from local raster data using a local geoprocessing package `.gpk` and the contour geoprocessing tool.

![Image of local server geoprocessing](LocalServerGeoprocessing.png)

## Use case

For executing offline geoprocessing tasks in your ArcGIS Runtime apps via an offline (local) server.

## How to use the sample

Contour Line Controls (Top Left):

* Interval - Specifies the spacing between contour lines.
* Generate Contours - Adds contour lines to map using interval.
* Clear Results - Removes contour lines from map.

## How it works

> 1. Create and run a local server with `LocalServer.instance`.

Reviewer: ```suggestion
1. Create and run a local server with `LocalServer.INSTANCE`.
```

2. Start the server asynchronously with `Server.startAsync()`.
3. Wait for server to be in the  `LocalServerStatus.STARTED` state.
   * Callbacks attached to `Server.addStatusChangedListener()` will invoke whenever the status of the local server has changed.
4. Start a `LocalGeoprocessingService` and run a `GeoprocessingTask`.
    1. Instantiate `LocalGeoprocessingService(Url, ServiceType)` to create a local geoprocessing service.
    2. Invoke `LocalGeoprocessingService.start()` to start the service asynchronously.
    3. Instantiate `GeoprocessingTask(LocalGeoprocessingService.url() + "/Contour")` to create a geoprocessing task that uses the contour lines tool.
5. Create an instance of `GeoprocessingParameters` and add a `GeoprocessingDouble` as a parameter using `setInterval`.
    1. Instantiate `GeoprocessingParameters(ExecutionType)` creates geoprocessing parameters.

>     2. Create a parameter using `inputs.insert("ContourInterval", new GeoprocessingDouble(double))` with name `ContourInterval` and with the interval set as its value.

Reviewer: reverting to Java specific instructions here
```suggestion
    2. Create a parameter using `GeoprocessingParameters.getInputs().put("Interval", new GeoprocessingDouble(double))` with name "Interval" and with the interval set as its value.
```

>     3.  Set the input with the interval value using `gpParams.setInputs(inputs)`.

Reviewer: looks like we don't use this step in Java, I can't find it in the code?

Author: You're right, this step is unnecessary. The action is already dealt with in the point above. I've removed it 👍
